lua/lint/linters/eugene.lua

```diff
@@ -0,0 +1,31 @@
+return {
+  cmd = 'eugene',
+  args = {
+    'lint',
+    '--format=json',
+  },
+  stdin = false,
+  parser = function(output, _)
+    local diagnostics = {}
+    if #output > 0 then
+      local ok, decoded = pcall(vim.json.decode, output)
+      if not ok then
+        return diagnostics
+      end
+      for _, stmt in ipairs(decoded.statements) do
+        if #stmt.triggered_rules > 0 then
+          for _, tr in ipairs(stmt.triggered_rules) do
+            table.insert(diagnostics, {
+              source = 'eugene',
+              lnum = stmt.line_number,
+              col = 0,
+              severity = vim.diagnostic.severity.ERROR,
+              message = string.format('%s: %s: %s', tr.id, tr.name, tr.url),
+            })
+          end
+        end
+      end
+    end
```

Comment:
```suggestion
        for _, tr in ipairs(stmt.triggered_rules) do
          table.insert(diagnostics, {
            source = 'eugene',
            lnum = stmt.line_number,
            col = 0,
            severity = vim.diagnostic.severity.ERROR,
            message = string.format('%s: %s: %s', tr.id, tr.name, tr.url),
          })
        end
      end
    end
```

The length check is kinda redundant